README.md

```diff
@@ -15,22 +15,36 @@ We have nightly and versioned Docker images, Debian packages, RPM packages, and
 
 This project requires Go 1.13 and Go module support.
 
-Set `GO111MODULE=on` or build the project outside of your `GOPATH` for it to succeed.
+Set `GO111MODULE=on` or build the project outside of your `GOPATH` for it to succeed. For information about modules, please refer to the [wiki](https://github.com/golang/go/wiki/Modules).
 
 The project also requires a recent stable version of Rust. We recommend using [rustup](https://rustup.rs/) to install Rust.
 
-If you are getting an `error loading module requirements` error with `bzr executable file not found in $PATH”` on `make`, then you need to ensure you have `bazaar`, `protobuf`, and `yarn` installed.
+You also need `make`, `bazaar`, `protobuf` and `yarn` installed.
 
 - OSX: `brew install bazaar protobuf yarn`
 - Linux (Arch): `pacman -S bzr protobuf yarn`
-- Linux (Ubuntu): `apt install bzr protobuf-compiler libprotobuf-dev yarnpkg`
+- Linux (Ubuntu, RHEL): See below
 
-**NB:** For RedHat, there are some extra steps:
+### Ubuntu-specific instructions:
+
+For Ubuntu, you need to fix the repository to fetch `yarn` from before using apt:
```

Comment:
Possibly better wording:

> For Ubuntu, you need to change the apt repository for `yarn`:

Reply:
Perhaps mention the two different yarns; a testing tool, and the one we use, so people know this isn't just a versioning difference, but a name conflict.

Reply:
:+1: to the suggested alternate wording